content-latest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+$args = array(
```

Comment:
Maybe we can think of creating a generic Wp_query function where we could add options to it on the fly as parameters and reuse it whenever it's needed a WP_query if that makes sense? 
I'm happy to help, do not intend to give you extra work :). I believe we all could benefit from it in the future.

Reply:
Yes, this is a reusable template part. The old blog theme is a mess, but I can't steer too far away from the design. Later we can look at a more unified design, so we can have one Wp_query function to cover them all. So I have two reusable template parts that almost do the same thing. This is fine for now. When we revisit the Blog for the redesign then these can be addressed.